Client/cypress/support/commands.ts

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Dit kan beter ergens anders. Bijv in een readme of een ander gedeelde document.

Reply:
Dit is de standaard documentatie van Cypress als het hier niet kan/mag dan gaat het weg en niet ergens anders heen. Dus alle code wordt verwijderd.